admin_guide/configuring_authentication.adoc

```diff
@@ -174,40 +192,78 @@ http://httpd.apache.org/docs/2.4/programs/htpasswd.html[`htpasswd`].
 
 === LDAP Authentication
 
-Set `*LDAPPasswordIdentityProvider*` in the `*identityProviders*` stanza to
-validate user names and passwords against an LDAP v3 server, using simple
-bind authentication.
-
-During authentication, the LDAP directory is searched for an entry that matches the provided username.
-If a single unique match is found, a simple bind is attempted using the distinguished name (DN) of the entry
-plus the provided password. Here are the steps taken:
+Set *LDAPPasswordIdentityProvider* in the `*identityProviders*` stanza to
+validate user names and passwords against an LDAPv3 server, using simple bind
+authentication.
+
+During authentication, the LDAP directory is searched for an entry that matches
+the provided user name. If a single unique match is found, a simple bind is
+attempted using the distinguished name (DN) of the entry plus the provided
+password. Here are the steps taken:
+
+. Generate a search filter by combining the attribute and filter in the
+configured `*url*` with the user-provided user name.
```

Comment:
They render properly in the docs.openshift and Customer Portal build:

https://docs.openshift.com/enterprise/3.0/admin_guide/configuring_authentication.html#AllowAllPasswordIdentityProvider

They only look dumb on the GH preview renders (but we can't rely on those anyway since they don't respect any of the ifdefs).

For "names of fields" I've been using the markup for "config params / env vars": https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#quick-markup-reference

For the _values_ of a field/param (as well as any generic "the name of this thing/object" e.g., a user name or the unique name of an API object), I've been using normal bold markup ("system item"?). Seemed best fit.